digitalocean/resource_digitalocean_kubernetes_cluster_test.go

```diff
@@ -281,6 +305,11 @@ func TestAccDigitalOceanKubernetesCluster_CreatePoolWithAutoScale(t *testing.T)
 							min_nodes = 1
 							max_nodes = 3
 						}
+						maintenance_policy {
+							start_time = "05:00"
+							duration = "04:00"
```

Comment:
Is a computed attribute (although the API documentation mentions that it can be said manually), so should be removed.

Reply:
@andrewsomething Except for this small thing, I think this PR is ready to be merged :partying_face: